app/poros/escape_facade.rb

```diff
@@ -2,14 +2,18 @@ class EscapeFacade
   def initialize(params, location)
     @params = params
     @location = location
-  end 
+  end
 
   def climb_options
     results = EscapeService.new.get_climb_results(@params, @location)
     results.map do |result|
-      if result[:type] == @params[:type] || result[:type] == "Sport, TR"
+      if result[:type] == @params[:trad] || result[:type] == "Sport, TR"
+        ClimbOptions.new(result)
+      elsif result[:type] == @params[:sport] || result[:type] == "Sport, TR"
+        ClimbOptions.new(result)
+      elsif !@params.key?(:trad) && !@params.key?(:sport)
```

Comment:
Would like to discuss this change and get clarification on why we are looking to instantiate ClimbOptions twice here.